services/frontend-service/src/legacy-ui/App/index.tsx

```diff
@@ -51,17 +51,53 @@ export const Spinner: React.FC<any> = (props: any) => {
     );
 };
 
+export const useInterval = (callback: () => void, delay: number) => {
+    const savedCallback = useRef();
+    useEffect(() => {
+        savedCallback.current = callback;
+    }, [callback]);
+    useEffect(() => {
+        function tick() {
+            if (savedCallback) {
+                savedCallback.current();
+            }
+        }
+        if (delay !== null) {
+            const id = setInterval(tick, delay);
+            return () => clearInterval(id);
+        }
+    }, [delay]);
+};
+
 const GetOverview = (props: { children: (r: api.GetOverviewResponse) => JSX.Element }): JSX.Element | null => {
+    // const [restart, setRestart] = useState(false);
+    const [count, setCount] = useState(0);
     const getOverview = React.useCallback(
-        (api: any, authHeader: AuthHeaderType) => api.overviewService().StreamOverview({}, authHeader),
-        []
+        (api: any) => api.overviewService().GetOverview({}),
+        // eslint-disable-next-line
+        [count]
     );
-    const overview = useObservable<api.GetOverviewResponse>(getOverview);
+    const overview = useUnary<api.GetOverviewResponse>(getOverview);
+    useInterval(() => {
+        setCount(count + 1);
+    }, 10 * 1000);
```

Comment:
10 sec is pretty often.
I think we should increase this to 20 or 30 and ensure that after a request (cart submit) GetOverview is called again.